src/chapter1/mac.md

```diff
@@ -1 +1,57 @@
 # Mac Setup
+
+For your training you will need a few tools. Namely:
+
+- Homebrew
+- Xcode
+- Git
+- C compiler toolchain (GCC)
+- A text editor (VSCode)
+
+## Installing Homebrew
+
+Homebrew is a package manager for MacOS. It offers thousands of developer libraries that would be difficult to obtain on the Mac platform. To install open the Terminal app and run the following command:
+
+```sh
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+
+[Homebrew](https://brew.sh/)
+
+## Installing Xcode
+
+Next we will need Xcode, Apple's compiler toolchain. Open the App Store and search for 'xcode' and press install.
+
```

Comment:
On macOS, `xcode-select --install` should install clang, gcc, and git without the use of brew. I don't think it requires the full installation of Xcode (humongous application) if you do it this way.

Reply:
Okay awesome, I do not have a Mac so I didn't know this. I'll adjust accordingly

Reply:
Do we pass arguments to the `xcode-select --install` like the apps we want to download or will it do it automatically?

Reply:
It's more of a standalone command which invokes a message whether to install a tool or not. If you run this command on a terminal, it should bring up a window similar to this:
![](https://mac.install.guide/assets/images/ruby/install-Xcode-CLT.png)
Clicking install will download and install the 3 aforementioned packages.